backend/models/discount.js

```diff
@@ -22,7 +22,7 @@ module.exports = (sequelize, DataTypes) => {
       discountType: {
         type: DataTypes.ENUM(DiscountTypeEnums)
       },
-      // If discountType is 'fixed', value is in dollar (not cents). Ex: 30 -> $30
+      // If discountType is 'fixed' or 'payment', value is in dollar (not cents). Ex: 30 -> $30
```

Comment:
I'm confused. I thought the automatic discounts were a percentage?

Reply:
Oh, yes, mistake on my end, will change this comment